tests/framework/objects/environment_manifest.go

```diff
@@ -41,7 +41,7 @@ func NewManifest() EnvironmentManifest {
 	flag.IntVar(&e.LoadConcurrentRuns, "load_parallel_runs", 20, "number of routines for load test")
 	flag.IntVar(&e.LoadVolumeNumber, "load_volumes", 1, "number of volumes(file,object or block) to be created for load test")
 	flag.IntVar(&e.LoadTime, "load_time", 1800, "number of seconds each thread perform operations for")
-	flag.Parse()
+	//flag.Parse()
```

Comment:
Nice, i'm glad the fix turned out to be this simple!! is there a reason to leave this as commented out?  Can we just delete it completely?